### PR TITLE
Refactor MTE-1402 [v118] [Onboarding]

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1402)

## :bulb: Description
Refactored onboarding automation tests

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme [hunk 1]
```diff
@@ -140,8 +140,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```
Comment (isabelrios) [inline Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme:143]: Could we undo these changes?

Comment (dragosb01) [reply Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme:143]: Not sure how this change got it, i will try to undo.

File: Tests/XCUITests/FirstRunTourTests.swift [hunk 2]
```diff
@@ -55,23 +56,36 @@ class FirstRunTourTests: BaseTestCase {
         waitForExistence(topSites)
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2250844
     func testCloseTour() {
```
Comment (isabelrios) [inline Tests/XCUITests/FirstRunTourTests.swift:60]: Nit: add // Smoketest before the title

File: Tests/XCUITests/FirstRunTourTests.swift [hunk 3]
```diff
@@ -55,23 +56,36 @@ class FirstRunTourTests: BaseTestCase {
...
     }
 
-    // MARK: Private
-    private func goToNextScreen() {
-        waitForExistence(app.buttons["\(rootA11yId)PrimaryButton"], timeout: 10)
-        app.buttons["\(rootA11yId)PrimaryButton"].tap()
-        currentScreen += 1
-    }
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/67227
+    func testWhatsNewPage() {
```
Comment (isabelrios) [inline Tests/XCUITests/FirstRunTourTests.swift:67]: I see this test enabled in the Smoketest but it is marked just as Functional... we should disable in the SmokeXCUItest test plan and enable in the Full Functional instead

File: Tests/XCUITests/FirstRunTourTests.swift (hunk 2 above)
Comment (isabelrios) [reply Tests/XCUITests/FirstRunTourTests.swift:60]: and please disable from the FullFuncional test plan

File: Tests/XCUITests/FirstRunTourTests.swift [hunk 4]
```diff
@@ -17,6 +17,7 @@ class FirstRunTourTests: BaseTestCase {
     }
 
     // Smoketest
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/471228
     func testFirstRunTour() {
```
Comment (isabelrios) [inline Tests/XCUITests/FirstRunTourTests.swift:21]: Please disable this from the FullFunctional

**Review by isabelrios — APPROVED** [review]
